sdk/core/core-http/lib/policies/userAgentPolicy.ts

```diff
@@ -15,6 +15,28 @@ import {
 
 export type TelemetryInfo = { key?: string; value?: string };
 
+/**
+ * Options for adding user agent details to outgoing requests.
+ */
+export interface UserAgentOptions {
+  /*
+   * String prefix to add to the user agent for outgoing requests.
+   * Defaults to an empty string.
+   */
+  userAgentPrefix?: string;
+
+  /**
+   * The name of the header in which the user agent string will be passed.
+   * Defaults to
+   */
+  userAgentHeaderName?: string;
```

Comment:
In what scenario do we want to allow users to customize the header name? Isn't this supposed to the be the same across all our Azure libraries as per the guidelines?

cc @bterlson

Reply:
This is gone now.